Android/app/src/main/java/use_case/kitchen/InventoryList.java

```diff
@@ -144,12 +144,14 @@ public static double getTotalQuantity(String name){
      * @param name The name of the ingredient being changed
      * @param usage the quantity used for this ingredient.
      */
-    public static void setQuantity(String name, double usage) {
+    public static String setQuantity(String name, double usage) {
+        String message;
         if (!myDict.containsKey(name)){
             //TODO: implement exceptions for cases of wrong key
-            return;
+            message = "wrong name";
         }
-        getItem(name).updateQuantity(usage);
+        message = getItem(name).updateQuantity(usage);
+        return message;
```

Comment:
see above comment about output boundary. Use case should call output boundary to change value in presenter